manticoresearch_text_embeddings.h

```diff
@@ -9,68 +9,48 @@
 #include <ostream>
 #include <new>
 
-using TextModelWrapper = void*;
+struct TextModelResult {
+  void *model;
```

Comment:
just to keep overall naming consistent:
`m_pModel` is more consistent in manticore style: m_ means 'member', p means 'pointer'